package.json

```diff
@@ -47,6 +47,9 @@
     "jose": "^1.27.2",
     "jsonwebtoken": "^8.5.1",
     "nodemailer": "^6.4.16",
+    "jwt-decode": "^2.2.0",
```

Comment:
`jwt-decode` has been removed, in favor of `jsonwebtoken`, it should be removed here as well.